.circleci/config.yml

```diff
@@ -48,14 +48,66 @@ jobs:
       # run go tests with gotestsum
       - run: |
           PACKAGE_NAMES=$(go list ./...)
-          gotestsum --format=short-verbose --junitfile $TEST_RESULTS/gotestsum-report.xml -- $PACKAGE_NAMES
+          gotestsum --format=short-verbose --junitfile $TEST_RESULTS/go-getter/gotestsum-report.xml -- $PACKAGE_NAMES
       - store_test_results:
           path: *TEST_RESULTS_PATH
       - store_artifacts:
           path: *TEST_RESULTS_PATH
 
+  windows-tests:
+    executor: win/default
+    working_directory: c:\gopath\src\github.com\hashicorp\go-getter
+    parameters:
+      go-version:
+        type: string
+    steps: 
+      - run: git config --global core.autocrlf false
+      - checkout
+
+      - restore_cache:
+          keys:
+            - win-gomod-cache-v1-{{ checksum "go.mod" }}
+
+      - run: go mod download
+
+      - save_cache:
+          key: win-gomod-cache-v1-{{ checksum "go.mod" }}
+          paths:
+            - c:\Windows\system32\config\systemprofile\go\pkg\mod
+
+      - run: 
+          name: Install dependencies
+          shell: bash.exe
+          command: |
+            mkdir -p /tmp/test-results/go-getter
+            # Module requires golang v.1.13+, but only 1.12 is pre-installed
+            choco install golang --version << parameters.go-version >>
+            curl -fOL https://github.com/gotestyourself/gotestsum/releases/download/v0.4.1/gotestsum_0.4.1_windows_amd64.tar.gz
+            tar -xvzf gotestsum_0.4.1_windows_amd64.tar.gz
+
+      - run:
+          name: Run tests with gotestsum 
+          shell: bash.exe
+          command: |
+            PACKAGE_NAMES=$(go list ./...)
+            ./gotestsum.exe --format=short-verbose --junitfile /tmp/test-results/go-getter/gotestsum-report.xml -- $PACKAGE_NAMES
+      - store_test_results:
+          path: c:\Users\circleci\AppData\Local\Temp\test-results
+      - store_artifacts:
+          path: c:\Users\circleci\AppData\Local\Temp\test-results
+
 workflows:
-  version: 2
-  test-and-build:
+  go-getter:
     jobs:
-      - go-fmt-and-test
+      - linux-tests:
+          context: go-getter
+          matrix:
+            parameters:
+              go-version: ["1.14.1"]
+          name: linux-test-go-<< matrix.go-version >>
+      - windows-tests:
+          context: go-getter
+          matrix:
```

Comment:
Nice! Not tried this feature yet :D

Reply:
I just found out about it recently! It's great  :)